packages/lesswrong/lib/collections/localgroups/views.ts

```diff
@@ -27,6 +27,17 @@ Localgroups.addDefaultView((terms: LocalgroupsViewTerms) => {
   };
 });
 
+Localgroups.addView("userActiveGroups", function (terms: LocalgroupsViewTerms) {
+  return {
+    selector: {
+      organizerIds: terms.userId,
```

Comment:
You *could* do this by selecting `organizerIds: context.currentUser._id` which would, (combined with some permissions stuff), allow anonymity of organizers. Up to you if you want to do that though, it's not obvious that we need it. If you do want to do it, the `addView` callback can take 3 arguments, the third of which is `context`.

Reply:
Interesting - thanks, this is an important consideration. I feel pretty unsure if people have privacy concerns about this, so I'm going to ask about it in the upcoming group organizers survey. I think it would be good to have a better sense of how public people want this association to be. I'll leave this as-is for now and will update it if necessary.

Also good to know about `context` for future reference, thanks! :)